scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementController.java

```diff
@@ -160,15 +128,56 @@ private void changeZValue(){
     double zValue = zValueSlider.getValue();
 
     // update the geometry of each of the existing graphics to include the new z-value
-    for (GraphicsOverlay graphicsOverlay : sceneView.getGraphicsOverlays()) {
-      for (Graphic graphic : graphicsOverlay.getGraphics()) {
+    sceneView.getGraphicsOverlays().forEach(graphicsOverlay ->
+      graphicsOverlay.getGraphics().stream().filter(
+        graphic -> graphic.getGeometry().getGeometryType() == GeometryType.POINT).forEach(graphic -> {
         Geometry geometry = graphic.getGeometry();
-        if (geometry.getGeometryType().equals(GeometryType.POINT)) {
-          Point currentPoint = (Point) geometry;
-          Point updatedPoint = new Point(currentPoint.getX(), currentPoint.getY(), zValue, currentPoint.getSpatialReference());
-          graphic.setGeometry(updatedPoint);
-        }
-      }
+        Point currentPoint = (Point) geometry;
+        Point updatedPoint = new Point(currentPoint.getX(), currentPoint.getY(), zValue, currentPoint.getSpatialReference());
+        graphic.setGeometry(updatedPoint);
+      }));
+  }
+
+  /**
+   * Creates a new TextSymbol from a string used to identify a surface placement type.
+   * @param text string to be used as text within the TextSymbol constructor
+   * @return a new TextSymbol
+   *
```

Comment:
again probably my fault for not spotting this in the suggestion! Can remove this empty javaDoc line. 
```suggestion
```